src/app/website/src/main.ml

```diff
@@ -531,7 +531,7 @@ let testnet () =
     ~extra_body:
       [ Html.literal
           {html|
-      <script defer src="https://s3-us-west-2.amazonaws.com/o1labs-snarkette-data/main.bc.js"></script>      
+      <script defer src="/static/main.bc.js"></script>
```

Comment:
I had to roll this back to using the main.bc.js that's hosted by our static server otherwise this shit